plugins/pencilblue/controllers/actions/admin/sites/new_site.js

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright (C) 2015  PencilBlue, LLC
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+module.exports = function NewSiteActionModule(pb) {
```

Comment:
We are trying to move away from that model because it creates a lot of
duplicate code. It is a nice to have rather than a must have.
On Jul 29, 2015 10:41 AM, "Andrew Parker" <notifications@github.com> wrote:

> In plugins/pencilblue/controllers/actions/admin/sites/new_site.js
> <https://github.com/pencilblue/pencilblue/pull/680#discussion_r35767099>:
>
> > +
> > + This program is free software: you can redistribute it and/or modify
> > + it under the terms of the GNU General Public License as published by
> > + the Free Software Foundation, either version 3 of the License, or
> > + (at your option) any later version.
> > +
> > + This program is distributed in the hope that it will be useful,
> > + but WITHOUT ANY WARRANTY; without even the implied warranty of
> > + MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > + GNU General Public License for more details.
> > +
> > + You should have received a copy of the GNU General Public License
> > + along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > + */
> > +
> > +module.exports = function NewSiteActionModule(pb) {
>
> I believe we opted for separate files for "edit site" and "new site" to
> maintain consistency with how the other admin actions (content, users) were
> constructed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pencilblue/pencilblue/pull/680/files#r35767099>.
>